dts/arm/nxp/nxp_rt118x_cm33.dtsi

```diff
@@ -30,8 +30,12 @@
 			ranges = <0x0 0x50000000 0x10000000>;
 		};
 
-		flexspi1: spi@525e0000 {
-			reg = <0x525e0000 0x4000>,<0x38000000 DT_SIZE_M(128)>;
+		flexspi: spi@525e0000 {
+			reg = <0x525e0000 0x4000>, <0x38000000 DT_SIZE_M(128)>;
+		};
+
+		flexspi2: spi@545E0000 {
```

Comment:
```suggestion
		flexspi2: spi@545e0000 {
```

Just a nit, but please be consistent within the file about the casing of hexadecimal addresses

Reply:
Updated.